csharp/test/Ice/adapterDeactivation/AllTests.cs

```diff
@@ -172,6 +172,7 @@ public static ITestIntfPrx Run(TestHelper helper)
                 IRouterPrx router = obj.Clone(IRouterPrx.Factory, label: "rc", identity: routerId);
                 {
                     using var adapter = communicator.CreateObjectAdapterWithRouter(router);
+                    adapter.Activate();
```

Comment:
I assume we plan to remove this `Activate` method in the near future since it's really just a wrapper around `ActivateAsync`. Should we just be calling `ActivateAsync` here?

Reply:
For a future PR.